tests/test_xopen.py

```diff
@@ -442,17 +523,17 @@ def test_write_pigz_threads_no_isal(tmpdir, xopen_without_igzip):
         assert f.read() == 'hello'
 
 
-def test_read_gzip_no_threads():
```

Comment:
Did you remove this by accident? The function below tests writing, this one tests reading.

Reply:
Yes, I did. I restored it and expanded the test.